.github/workflows/minimal-app.yml

```diff
@@ -37,7 +43,8 @@ jobs:
         name: build image
 
   release:
-    if: github.ref == 'refs/heads/main'
+#    if: github.ref == 'refs/heads/main'
```

Comment:
❓ Should this be uncommented?

Reply:
Sorry moving things a little bit under your feet. Confirmed that releases work so I'll remove that in just a sec.

Reply:
Removed